02-standard-library/activityMonitor.ts

```diff
@@ -0,0 +1,41 @@
+import { exec } from "child_process";
```

Comment:
please use the aforementioned `node:...` prefix for in-built modules.

https://2ality.com/2021/12/node-protocol-imports.html